tests/Rector/Class_/StaticDataProviderClassMethodRector/Fixture/test_case.php.inc

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace PHPUnit\Framework;
+
+namespace Rector\Tests\PHPUnit\Rector\Class_\StaticDataProviderClassMethodRector\Fixture;
+
+class TestCase
```

Comment:
Not sure this works, I think you check for this namespace in rules, right?

Reply:
Yeah, it should be spit 1 class = 1 file, so PHPStan can autoload it propperly.
But don't worry about it :)

Reply:
Would be cool if one could "Add new file" like for gists.github.com on the demo page

Reply:
I think that might lead people to commiting /vendor here :D 

Also, does Github link allow multiple files?

Reply:
> Also, does Github link allow multiple files?

I don't know